floater/generators.py

```diff
@@ -232,6 +233,8 @@ def to_mitgcm_format(self, filename, tstart=0, iup=0, mesh='rect', model_grid=No
         # which was was wrong for masked cases
         N = len(lon)
 
+        self.N = N 
```

Comment:
I do not like this. These methods should not modify the class attributes. If you want to set ``self.N``, set it in ``__init__``.